### ISSUE TITLE
Not able to install in a .NET MAUI project

### ISSUE THREAD

**Issue by andreas-nesheim** [TobiasBuchholz/Plugin.Firebase#65]
I get the following error when I try to install the latest version (1.2.0) into a .NET MAUI project:

Could not find a part of the path 'C:\Users\andre\.nuget\packages\xamarin.firebase.ios.installations\8.10.0.1\lib\net6.0-ios15.4\Firebase.Installations.resources\FirebaseInstallations.xcframework\ios-arm64_i386_x86_64-simulator\FirebaseInstallations.framework\Headers\FirebaseInstallations-umbrella.h'.				

This can be reproduced by creating a blank .NET MAUI app in Visual Studio version 17.3.3.

**Comment by andreas-nesheim** [comment]
I tested it on VS for Mac and it worked fine there. A friend of mine mentioned that this could be the MAX_PATH on Windows kicking in and suggested that I enabled Long Path support: https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-long-path

I tried this, but it still didn't seem to work. The article mentioned that the dotnet CLI already supports long paths, so I tried adding the package with the dotnet CLI using `dotnet add package Plugin.Firebase`. That did the trick. Just a heads up for others who might get the same error.
